### ISSUE TITLE
Project deprecated?

### ISSUE THREAD

**Issue by cabiad** [tophat/codewatch#108]
@lime-green / @rohitjain 

I've been focusing on other things than this project from a while now and haven't seen many changes coming though from other contributors, including for the auto-detected security issues.

I'm curious about your thoughts on whether it's time to mark this project as deprecated.

**Comment by lime-green** [comment]
@cabiad yeah I think it makes sense. Are you thinking something like a README update, or is there a more official way to do so?

**Comment by cabiad** [comment]
A deprecation notice seems like a good start. The other alternative is to archive the project through the settings. I'm open to either.

For now, I put together a quick PR for deprecation, listing some alternatives if people find this project organically. https://github.com/tophat/codewatch/pull/110
